.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,76 @@
+# Contributing to Destination Sol
+
+If you would like to contribute code, documentation, or other assets you can do so through GitHub by forking the repository and sending a pull request (PR). You can also simply report issues (bugs), but please search for any previous reports first.
+
+*When submitting code, please make every effort to follow existing conventions and style in order to keep the code as readable as possible.*
+
+Read on for an overview and [check the wiki for more details](https://github.com/MovingBlocks/DestinationSol/wiki). For questions please join us in our [forum](http://forum.terasology.org/forum) or on `#destination-sol` (irc.freenode.net).
+
```

Comment:
Can we please change `#destionation-sol` to `#terasology`? That makes communication way easier for everyone, since not many people frequent #destSol (apart from cervator, of course :D )